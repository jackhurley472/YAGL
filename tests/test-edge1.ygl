Graph G;
Node n1("origin");
Node n2("destination");
G + n1;
G + n2;
G: n1 ->5 n2;
printGraph(G);
