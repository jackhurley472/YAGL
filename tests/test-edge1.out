============== Graph Print ===============
	Stats: 	2	2	1

Nodes:
Node (0) : origin --- Node (1) : destination

Edges:
[(0) : origin] ---(5)---> [(1) : destination]
============ End Graph Print =============
